frontends/infinite-corridor/src/pages/NotFoundPage.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import Container from "@mui/material/Container"
+import Card from "@mui/material/Card"
+import CardContent from "@mui/material/CardContent"
+import CardActions from "@mui/material/CardActions"
+import Button from "@mui/material/Button"
+
+import { HOME } from "./urls"
+import { MetaTags } from "ol-util"
+
+const NotFoundPage: React.FC = () => {
+  return (
+    <Container maxWidth="sm">
+      <Card sx={{ marginTop: "1rem" }}>
+        <MetaTags>
+          <meta name="robots" content="noindex,noarchive" />
+        </MetaTags>
+        <CardContent>
+          <h1>404 Not Found Error</h1>
+          Resource Not Found
+        </CardContent>
+        <CardActions>
+          <Button aria-label="Return Home" variant="outlined" href={HOME}>
+            Home
+          </Button>
```

Comment:
_Not requesting a change!_

**Comment:** The `aria-label` is fine, but IMO not necessary.
- with aria-label, screenreaders will read this as something like "Link: Return Home"
- without, "Link: Home"

"Link: Home" seems pretty intelligible to me.